staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/versionconverter.go

```diff
@@ -99,3 +101,23 @@ func (v *versionConverter) Convert(object *typed.TypedValue, version fieldpath.A
 func (v *versionConverter) IsMissingVersionError(err error) bool {
 	return runtime.IsNotRegisteredError(err) || isNoCorrespondingTypeError(err)
 }
+
+type noCorrespondingTypeErr struct {
```

Comment:
Does it make sense for this error type to be private? Given that it is returned by a public function? That makes it impossible to use `errors.As` correct?

Reply:
Note that this was private already, this is using the pattern described in this articl: https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully
This is mostly unrelated to the change, I just moved this.
/shrug